packages/e2e-tests/kit-node/vite.config.js

```diff
@@ -15,11 +15,33 @@ const config = {
 		minify: false,
 		sourcemap: true // must be true for hermetic build test!
 	},
-	plugins: [transformValidation(), sveltekit(), writeResolvedConfig()],
+	plugins: [
+		transformValidation(),
+		sveltekit(),
+		writeResolvedConfig(),
+		workaroundInlineSvelteCssIssue()
+	],
 	optimizeDeps: {
 		// eagerly include these, otherwise vite optimizer might interfere with restarting while the test is running
 		include: ['svelte-i18n', 'e2e-test-dep-svelte-api-only']
 	}
 };
 
-export default config;
+/**
+ * Workaround until https://github.com/sveltejs/kit/pull/13007 is merged
```

Comment:
lets merge the kit PR first then, really not a fan of extra code in tests to work around issues in the setup being tested. users don't have the same luxury so this is just hiding an issue. 

would rather merge this with a failed test than hiding it if we can't get the kit PR released in time

Reply:
I don't think it's hiding since there's already a PR out to track this. If it's still not getting reviews I don't think we should be sacrificing our CI for it.

Reply:
added an issue to remind us to remove this